modules/lambdas/main.tf

```diff
@@ -479,3 +479,300 @@ data "aws_lambda_invocation" "db_migration" {
 }
 
 ## TODO: Should create null resource to handle password changes ORCA-145
+
+# API Gateway- API for orca_catalog_reporting lambda function
+resource "aws_api_gateway_rest_api" "orca_catalog_reporting_api" {
+  name        = "${var.prefix}_orca_catalog_reporting_api"
+  description = "API for catalog reporting lambda function"
+  endpoint_configuration {
+    types = ["PRIVATE"]
+    # Cumulus might need to create vpc_endpoint_ids which should be added here in the future in order to access this API
+  }
+}
+resource "aws_api_gateway_resource" "orca_catalog_reporting_api_resource" {
+  path_part   = "{proxy+}"
```

Comment:
This looks odd to me. shouldn't this be "resource"?

Reply:
not  sure about this. This shows as {proxy+} according to @AndrewDorn PR https://github.com/nasa/cumulus-orca/pull/102

Reply:
May want to do more investigation. I think proxy is a catchall and maybe not what we want to do here for each one.

Reply:
changed the {proxy} to `orca_catalog_reporting`, `recovery_status_granules` and `recovery_status_jobs`

tried using / instead of _ but that is not allowed in the format standards.